tools/backend/lsq-generator-python/vhdl_gen/codegen.py

```diff
@@ -0,0 +1,35 @@
+from vhdl_gen.context import VHDLContext
+from vhdl_gen.generators import *

```

Comment:
same here as with the namespacing in main, import the library as a variable (with snake case name) and then call these functions from it :)

Reply:
```suggestion
import vhdl_gen.generators.group_allocator as group_allocator
import vhdl_gen.generators.dispatchers as dispatchers
import vhdl_gen.generators.lsq as lsq
```
Sure, I changed it this way